prisma/schema.prisma

```diff
@@ -1025,3 +1027,19 @@ enum chat_type {
   NORMAL
   ANNOUNCEMENT
 }
+
+enum instructor_screening_jobstatus_enum {
```

Comment:
Also prisma isnt really a fan of umlaute.
![image](https://github.com/corona-school/backend/assets/48623649/670cee60-73f0-45a4-aa47-57d928ebab17)
Not sure how if this will be display correctly in retool

Reply:
I would prefer if we would keep the enums english, translation can be done in the UI